doc/layers.md

```diff
@@ -0,0 +1,92 @@
+# Layers
+
+Layers are an useful feature that lets you group renderization by context, and to also to obtain a way of pre rendering things, enabling for example the renderization in memory of parts of your game that don't change much, like a background for example, and by doing that, freeing resources for more dynamic content that needs to be rendered every loop cycle.
+
+There are two types of layers on Flame: `DynamicLayer` and `PreRenderedLayer`.
```

Comment:
I think this sentence would be improved with a very brief description of each, ie:

There are two types of layers on Flame: `DynamicLayer` (for things that are moving or changing) and `PreRenderedLayer` (for things that are static).